pyproject.toml

```diff
@@ -30,6 +30,9 @@ default = "goose.profile:default_profile"
 [project.entry-points."goose.command"]
 file = "goose.command.file:FileCommand"
 
+[project.entry-points."metadata.plugins"]
+goose-ai = ""
```

Comment:
Take it doing something like package_name="goose-ai" didn't work?

Reply:
I just realized the exports probably would overwrite each other for that case

Reply:
Yes, I think you’re right! Good point